### PR TITLE
Replace datetime(calendar) widget from controls1

### PR DESCRIPTION
**Pull request by tomasMizera**
Previous calendar used `quick controls 1`. We thus needed to replace it with new one.

New calendar look:

https://user-images.githubusercontent.com/22449698/145367793-37b15f82-a3ca-475e-b850-4fa6be31fd76.mp4

@jozef-budac if you want, I created project `tomas/test_datetime`(dev) that contains all different combinations of datetime widgets.

### CODE REVIEW

File: app/qml/components/DateTimePicker.qml [hunk 1]
```diff
@@ -0,0 +1,628 @@
+import QtQuick 2.14
```
Comment (PeterPetrik) [inline app/qml/components/DateTimePicker.qml:1]: missing copyright

File: app/qml/components/DateTimePicker.qml [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+import Qt.labs.calendar 1.0
+import QtGraphicalEffects 1.14
+
+// Inspired by https://gist.github.com/ilnuribat/5cc6010aea7e35708ff90c65d7be5bcf
```
Comment (PeterPetrik) [inline app/qml/components/DateTimePicker.qml:7]: remove, this is very little with the origin code

File: app/qml/components/DateTimePicker.qml [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+      event.accepted = true
+      root.canceled()
+    }
+  }
+
+  height: 400
+  width: 400
+
+  QtObject {
+    id: palette
```
Comment (PeterPetrik) [inline app/qml/components/DateTimePicker.qml:69]: I would prefer to use white color for back and save, or do we have this orange on green somewhere else?

File: app/qml/components/DateTimePicker.qml [hunk 4]
```diff
@@ -0,0 +1,628 @@
...
+        }
+
+        color: palette.primary_dark
+        Text {
+          id: cancelBtnText
+          anchors.centerIn: parent
+          font.pointSize: root.fontPointSizeBig
+          font.bold: true
+          color: "#FD9626"
+          text: "Cancel"
```
Comment (wonder-sk) [inline app/qml/components/DateTimePicker.qml:114]: qsTr

File: app/qml/components/DateTimePicker.qml [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+        Text {
+          id: okBtnText
+
+          anchors.centerIn: parent
+
+          font.pointSize: root.fontPointSizeBig
+          font.bold: true
+
+          color: "#FD9626"
+          text: "Ok"
```
Comment (wonder-sk) [inline app/qml/components/DateTimePicker.qml:207]: qsTr + maybe use uppercase OK ?

**Review by wonder-sk — COMMENTED** [review]
just minor bits I picked from the video, didn't do a proper code review...

**Review by PeterPetrik — APPROVED** [review]

File: app/qml/components/DateTimePicker.qml (hunk 3 above)
Comment (tomasMizera) [reply app/qml/components/DateTimePicker.qml]: I am not sure about this, I wanted to let users know that these areas are clickable

**Comment by tomasMizera** [comment]
@PeterPetrik ios:

**Pull request merged**